packages/core/content-manager/server/src/controllers/collection-types.ts

```diff
@@ -209,7 +209,7 @@ export default {
         permissionChecker,
         model,
         // @ts-expect-error TODO: fix
-        { id, locale, publishedAt: null },
+        { documentId: id, locale, publishedAt: null },
```

Comment:
was this working without the document id 😮

Reply:
Not as expected, for this reason it was returning always empty `availableLocales`. Probably we don't noticed it because this case only happen when you are creating a new entry and we don't need the availableLocales when creating for other thing different to fill in from another locale 🤔

Reply:
dont we need it to list the available locales in the list of the document locales? I thought that was working before

Reply:
could we add some tests for that ? 🙏 in the content manager api tests